README.md

```diff
@@ -50,10 +47,26 @@ Version Control (Git)
 
 ### Installation
 
-* After cloning the repository, in the project directory, `cd` into hackton-frontend and you can run: `npm install` or `yarn install` in the console.
-* Then run `npm start` or `yarn start`.
+* Start by cloning the repository to your local machine
+
+```bash
+git clone https://github.com/Labs-EU4/hackathon-portal-client.git
+```
+
+* Next install the package dependencies by running; this can be `npm` or `yarn
+
+```bash
+npm install
+```
+
+* After that start or build the application by running `npm start` or `yarn start`
+
+```bash
+yarn start # For a local development
```

Comment:
I would make this `npm start` in order to have consistency with the other two command above

Reply:
Completed